python/streamsim/sender.py

```diff
@@ -0,0 +1,25 @@
+import logging
+import confluent_kafka
+
+
+class AlertProducer(confluent_kafka.Producer):
+    """A client capable of sending alert data to a Kafka broker.
+
+    """
+    def __init__(self, broker_url, topic):
+        kafka_config = {
+            "bootstrap.servers": broker_url,
+        }
+        self.topic = topic
+        super().__init__(kafka_config)
+
+    def send_alert(self, alert):
+        """Send a single alert packet to the Kafka topic.
+
+        Parameters
+        ----------
+        alert : `bytes`
+            A sequence of bytes that should encode an alert.
+        """
+        logging.debug("sending alert (%d bytes)", len(alert))
+        self.produce(topic=self.topic, value=alert)
```

Comment:
Obviously you know this, but this doesn't seem to do the whole “Confluent Wire Format” thing (unless the file already has magic bytes prepended). I don't think that's necessarily a problem (maybe you plan to add it at a later stage, or maybe you don't think it's useful, either of which is fine), but since it's at variance with DMTN-093 it might be useful to just mention in a comment what's going on.

Reply:
Yep, totally right. Proper alert formatting is to come in a future commit.

Reply:
Created https://jira.lsstcorp.org/browse/DM-24681 to track this.